art/command_line.py

```diff
@@ -98,8 +115,12 @@ def update():
     """Update latest tag/branch job IDs."""
 
     gitlab = get_gitlab()
-    artifacts = _yaml.load(_paths.artifacts_file)
 
+    # As of GitLab 16.4, you cannot access the projects and jobs APIs with a job token
+    if is_using_job_token(gitlab):
+        raise Exception('A job token cannot be used to update artifacts')
```

Comment:
Consider raising a Click [exception](https://click.palletsprojects.com/en/8.0.x/api/#exceptions) type here to avoid a backtrace. E.g. `UsageError`

Reply:
All current exceptions are derived from `ClickException` in fa6044fbda8437deb5be389a2e72c268286df5d7